sklearn/datasets/tests/test_openml.py

```diff
@@ -10,6 +10,7 @@
 import sklearn
 import pytest
 
+from io import BytesIO
```

Comment:
See comment above.

Reply:
Let's reorganize properly the import as suggested by @lorentzenchr: the convention is standard libraries, then 3rd-party libraries and local files
 
```python
import gzip
import json
import os
import re
from io import BytesIO

import numpy as np
import scipy.sparse
import sklearn
import pytest
```

Reply:
Fixed!